sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Release History
 
 ## 1.2.0 (Unreleased)
-
 ### Features Added
+- Exported types `MicrosoftTeamsUserIdentifier`, `PhoneNumberIdentifier`, `UnknownIdentifier` for non Azure Communication Services `CommunicationIdentifier` identities.
```

Comment:
Note to reviewers: Please focus on changes in `azure-communication-identity` first. The changes in other packages are just copies and have no API implications for those packages.